btree.go

```diff
@@ -181,13 +181,27 @@ func (q *x) siblings(i int) (l, r *d) {
 func (l *d) mvL(r *d, c int) {
 	copy(l.d[l.c:], r.d[:c])
 	copy(r.d[:], r.d[c:r.c])
+	// Technically, zeroing out the de's here is not necessary,
```

Comment:
This is kinda spooky... is it needed or not? If not, let's delete the code.

Reply:
> // Technically, zeroing out the de's here is not necessary ...

Technically, it's essential to zero out invalid item as they may reference otherwise unreachable stuff that would otherwise become non-collectable from the garbage collector point of view.

Reply:
Ah, good point @cznic. Thanks. I'll uncomment this code and run some benchmarks against it.